modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -0,0 +1,204 @@
+{{#hasHttpSignatureMethods}}
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+from six.moves.urllib.parse import urlencode, urlparse
+from Crypto.PublicKey import RSA, ECC
+from Crypto.Signature import PKCS1_v1_5, pss, DSS
+from Crypto.Hash import SHA256, SHA512
+from base64 import b64encode
+from email.utils import formatdate
+
+class HttpSignatureHandler(object):
+    """OpenAPI HTTP signature handler to generate and verify HTTP signatures.
+
+    NOTE: This class is auto generated by OpenAPI Generator.
+    Ref: https://openapi-generator.tech
+    Do not edit the class manually.
+
+    :param configuration: .Configuration object for this client
+    """
+
+    ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
+    ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
+    ECDSA_KEY_SIGNING_ALGORITHMS = {ECDSA_MODE_FIPS_186_3, ECDSA_MODE_DETERMINISTIC_RFC6979}
+
+    def __init__(self, configuration=None):
```

Comment:
How about deleting this class?
We only use it to store the configuration object which could be passed in to get_http_signature_headers.

Reply:
done.